nologs/nologs.py

```diff
@@ -0,0 +1,18 @@
+"""
+Use this tool in conjunction with the following code in your primary C file (following #include <pebble.h>) to
+include/exclude logs based on whether a build is passed the --release flag:
```

Comment:
The docstring says the flag is `--release`, but the option in the code is named `--no-log`.